source/reference/minio-mc-admin/mc-admin-scanner-status.rst

```diff
@@ -0,0 +1,129 @@
+===========================
+``mc admin scanner status``
+===========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin scanner info
+.. mc:: mc admin scanner status
+
+Description
+-----------
+
+.. start-mc-admin-scanner-status-desc
+
+The :mc-cmd:`mc admin scanner status` command displays a real-time summary of :ref:`scanner <minio-concepts-scanner>` information for a MinIO Server.
+
+.. end-mc-admin-scanner-status-desc
+
+This command has also be run by using ``mc admin scanner info``.
```

Comment:
```suggestion
This command has an alias of ``mc admin scanner info``.
```

Reply:
I was attempting to avoid `alias`, since we use it for a specific meaning within the MinIO context. But we do already use alias this way for other commands, just going with it.